pkg/engine/engine.go

```diff
@@ -158,6 +161,23 @@ func (e *Engine) alterFuncMap(t *template.Template) template.FuncMap {
 		return val, nil
 	}
 
+	// Add the 'tpl' function here
+	funcMap["tpl"] = func(tpl string, vals chartutil.Values) string {
+		r := renderable{
+			tpl:  tpl,
+			vals: vals,
+		}
+
+		templates := map[string]renderable{}
+		templates["template"] = r
+
+		result, err := e.render(templates)
+		if err != nil {
```

Comment:
Thanks for the input. I changed the function to return an error on failure instead of writing it into the template.